pkg/opcfg/config.go

```diff
@@ -375,3 +375,13 @@ func getStackTrace() zap.Option {
 	}
 	return zap.AddStacktrace(zapcore.LevelEnabler(lvl))
 }
+
+// GetVdbMaxBackoffDuration returns maximum backoff requeue duration in milliseconds for vdb controller
+func GetVdbMaxBackoffDuration() int {
+	duration := lookupIntEnvVar("VDB_MAX_BACKOFF_DURATION", envCanNotExist)
+	if duration == 0 {
```

Comment:
If we set the value VDB_MAX_BACKOFF_DURATION=0, should we expect 0ms duration?

Reply:
Yes, we do expect 0ms duration. I will change this one to "<0".